ui/src/app/views/account-creation/component.ts

```diff
@@ -35,6 +35,7 @@ export class AccountCreationComponent implements AfterViewInit {
   gsuiteDomain: string;
   usernameOffFocus = true;
```

Comment:
Do you still need this variable, in that case? 

nit: I think this does go back to my original pet peeve on this feature which is a red mark being shown as you enter your name.